README.md

```diff
@@ -38,6 +38,28 @@ A: No. You can, but running `bundle exec zeus` instead of `zeus` adds precious s
 It is common to see tests running twice when starting out with Zeus. If you see your tests/specs running twice, you should try disabling `require 'rspec/autotest'` and `require 'rspec/autorun'` (for RSpec), or `require 'minitest/autorun'` (for Minitest). (see [#134](https://github.com/burke/zeus/issues/134) for more information).
 
 
+## Set up 
+
+In your Rails app's directory initialize zeus:
+
+    zeus init
+
+which will create two files that should be added to your `.gitignore` (or globally in `~/.gitignore`)
+
+    custom_plan.rb
+    zeus.json
+
```

Comment:
I don't quite understand the recommendation to `gitignore` those files? If the project uses zeus it probably makes sense to share those, if it is only one developer trying it out probably ignore them. Am I missing something? (I have so far never ignored them thats why I'm asking :) )